drivers/hsc/hsc.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * 
+ */
+
+/**
+ * @ingroup     drivers_hsc
```

Comment:
Surely the part is not just called HSC - does it have a part number?

Reply:
This driver can be use with the whole HSC series. For different part number, only some parameters need to be changed.

Reply:
![HSC_parameters](https://user-images.githubusercontent.com/32953667/98361900-684e7c00-203d-11eb-9674-1662ea45a852.png)